### ISSUE TITLE
Class not found in the raindropio module.

### ISSUE THREAD

**Issue by drillan** [atsuoishimoto/python-raindropio#1]
I tried python-raindropio==0.0.1 from [python:3.8-slim Docker image](https://hub.docker.com/layers/python/library/python/3.8-slim/images/sha256-9b574c348d20673be5a5c716f4fdefac56643c103c0c7005e2160c6a843faab8?context=explore).

```
root@d825f7f390a3:/# pip install python-raindropio
Collecting python-raindropio
  Downloading python_raindropio-0.0.1-py3-none-any.whl (4.3 kB)
Collecting python-dotenv
  Downloading python_dotenv-0.13.0-py2.py3-none-any.whl (17 kB)
Collecting requests
  Downloading requests-2.24.0-py2.py3-none-any.whl (61 kB)
     |████████████████████████████████| 61 kB 200 kB/s 
Collecting python-dateutil
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 477 kB/s 
Collecting chardet<4,>=3.0.2
  Downloading chardet-3.0.4-py2.py3-none-any.whl (133 kB)
     |████████████████████████████████| 133 kB 457 kB/s 
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
  Downloading urllib3-1.25.9-py2.py3-none-any.whl (126 kB)
     |████████████████████████████████| 126 kB 306 kB/s 
Collecting certifi>=2017.4.17
  Downloading certifi-2020.4.5.2-py2.py3-none-any.whl (157 kB)
     |████████████████████████████████| 157 kB 296 kB/s 
Collecting idna<3,>=2.5
  Downloading idna-2.9-py2.py3-none-any.whl (58 kB)
     |████████████████████████████████| 58 kB 223 kB/s 
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Installing collected packages: python-dotenv, chardet, urllib3, certifi, idna, requests, six, python-dateutil, python-raindropio
Successfully installed certifi-2020.4.5.2 chardet-3.0.4 idna-2.9 python-dateutil-2.8.1 python-dotenv-0.13.0 python-raindropio-0.0.1 requests-2.24.0 six-1.15.0 urllib3-1.25.9
root@d825f7f390a3:/# pip freeze
certifi==2020.4.5.2
chardet==3.0.4
idna==2.9
python-dateutil==2.8.1
python-dotenv==0.13.0
python-raindropio==0.0.1
requests==2.24.0
six==1.15.0
urllib3==1.25.9
root@d825f7f390a3:/# python
Python 3.8.3 (default, Jun  9 2020, 17:49:41) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import raindropio
>>> dir(raindropio)
['__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__']
>>> 
```

**Comment by drillan** [comment]
I installed it from a GitHub repository and tried it, and it was fine.

```
root@d825f7f390a3:/# git clone https://github.com/atsuoishimoto/python-raindropio.git
Cloning into 'python-raindropio'...
remote: Enumerating objects: 118, done.
remote: Counting objects: 100% (118/118), done.
remote: Compressing objects: 100% (68/68), done.
remote: Total 118 (delta 56), reused 109 (delta 47), pack-reused 0
Receiving objects: 100% (118/118), 27.61 KiB | 247.00 KiB/s, done.
Resolving deltas: 100% (56/56), done.
root@d825f7f390a3:/# pip install python-raindropio/
Processing /python-raindropio
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
    Preparing wheel metadata ... done
Requirement already satisfied: requests in /usr/local/lib/python3.8/site-packages (from python-raindropio==0.0.3) (2.24.0)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.8/site-packages (from python-raindropio==0.0.3) (2.8.1)
Requirement already satisfied: python-dotenv in /usr/local/lib/python3.8/site-packages (from python-raindropio==0.0.3) (0.13.0)
Collecting jashin
  Downloading jashin-0.0.5-py3-none-any.whl (7.5 kB)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.8/site-packages (from requests->python-raindropio==0.0.3) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.8/site-packages (from requests->python-raindropio==0.0.3) (1.25.9)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.8/site-packages (from requests->python-raindropio==0.0.3) (2020.4.5.2)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.8/site-packages (from requests->python-raindropio==0.0.3) (2.9)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/site-packages (from python-dateutil->python-raindropio==0.0.3) (1.15.0)
Building wheels for collected packages: python-raindropio
  Building wheel for python-raindropio (PEP 517) ... done
  Created wheel for python-raindropio: filename=python_raindropio-0.0.3-py3-none-any.whl size=6331 sha256=b43ae1f94c545ee9347550552ab5c5f6c978f84953cb6ffed5ba88c9e3d0b4c5
  Stored in directory: /root/.cache/pip/wheels/e5/ab/93/a9adf3358aadd22e14ff10d9ae3f6def4a5ea36ba5df22d0f3
Successfully built python-raindropio
Installing collected packages: jashin, python-raindropio
  Attempting uninstall: python-raindropio
    Found existing installation: python-raindropio 0.0.1
    Uninstalling python-raindropio-0.0.1:
      Successfully uninstalled python-raindropio-0.0.1
Successfully installed jashin-0.0.5 python-raindropio-0.0.3
root@d825f7f390a3:/# pip freeze
certifi==2020.4.5.2
chardet==3.0.4
idna==2.9
jashin==0.0.5
python-dateutil==2.8.1
python-dotenv==0.13.0
python-raindropio @ file:///python-raindropio
requests==2.24.0
six==1.15.0
urllib3==1.25.9
root@d825f7f390a3:/# python
Python 3.8.3 (default, Jun  9 2020, 17:49:41) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import raindropio
>>> dir(raindropio)
['API', 'Access', 'AccessLevel', 'BrokenLevel', 'Collection', 'CollectionRef', 'DictModel', 'FontColor', 'Group', 'Raindrop', 'RaindropType', 'User', 'UserConfig', 'UserFiles', 'UserRef', 'View', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__', 'api', 'models']
>>> 
```

**Comment by drillan** [comment]
I built and installed the wheel myself from a new container. Then it worked fine.

```
root@cf227dae855d:/python-raindropio# python setup.py bdist_wheel
running bdist_wheel
running build
running build_py
creating build
creating build/lib
creating build/lib/raindropio
copying raindropio/__init__.py -> build/lib/raindropio
copying raindropio/api.py -> build/lib/raindropio
copying raindropio/models.py -> build/lib/raindropio
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/raindropio
copying build/lib/raindropio/__init__.py -> build/bdist.linux-x86_64/wheel/raindropio
copying build/lib/raindropio/api.py -> build/bdist.linux-x86_64/wheel/raindropio
copying build/lib/raindropio/models.py -> build/bdist.linux-x86_64/wheel/raindropio
running install_egg_info
running egg_info
creating python_raindropio.egg-info
writing python_raindropio.egg-info/PKG-INFO
writing dependency_links to python_raindropio.egg-info/dependency_links.txt
writing requirements to python_raindropio.egg-info/requires.txt
writing top-level names to python_raindropio.egg-info/top_level.txt
writing manifest file 'python_raindropio.egg-info/SOURCES.txt'
writing manifest file 'python_raindropio.egg-info/SOURCES.txt'
Copying python_raindropio.egg-info to build/bdist.linux-x86_64/wheel/python_raindropio-0.0.3-py3.8.egg-info
running install_scripts
adding license file "LICENSE" (matched pattern "LICEN[CS]E*")
creating build/bdist.linux-x86_64/wheel/python_raindropio-0.0.3.dist-info/WHEEL
creating 'dist/python_raindropio-0.0.3-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'raindropio/__init__.py'
adding 'raindropio/api.py'
adding 'raindropio/models.py'
adding 'python_raindropio-0.0.3.dist-info/LICENSE'
adding 'python_raindropio-0.0.3.dist-info/METADATA'
adding 'python_raindropio-0.0.3.dist-info/WHEEL'
adding 'python_raindropio-0.0.3.dist-info/top_level.txt'
adding 'python_raindropio-0.0.3.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
root@cf227dae855d:/python-raindropio# cd ..
root@cf227dae855d:/# pip install python-raindropio/dist/python_raindropio-0.0.3-py3-none-any.whl 
Processing /python-raindropio/dist/python_raindropio-0.0.3-py3-none-any.whl
Collecting jashin
  Downloading jashin-0.0.5-py3-none-any.whl (7.5 kB)
Collecting python-dateutil
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 11.5 MB/s 
Collecting python-dotenv
  Downloading python_dotenv-0.13.0-py2.py3-none-any.whl (17 kB)
Collecting requests
  Downloading requests-2.24.0-py2.py3-none-any.whl (61 kB)
     |████████████████████████████████| 61 kB 273 kB/s 
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2020.4.5.2-py2.py3-none-any.whl (157 kB)
     |████████████████████████████████| 157 kB 17.2 MB/s 
Collecting idna<3,>=2.5
  Downloading idna-2.9-py2.py3-none-any.whl (58 kB)
     |████████████████████████████████| 58 kB 6.8 MB/s 
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
  Downloading urllib3-1.25.9-py2.py3-none-any.whl (126 kB)
     |████████████████████████████████| 126 kB 13.9 MB/s 
Collecting chardet<4,>=3.0.2
  Downloading chardet-3.0.4-py2.py3-none-any.whl (133 kB)
     |████████████████████████████████| 133 kB 17.8 MB/s 
Installing collected packages: certifi, idna, urllib3, chardet, requests, jashin, six, python-dateutil, python-dotenv, python-raindropio
Successfully installed certifi-2020.4.5.2 chardet-3.0.4 idna-2.9 jashin-0.0.5 python-dateutil-2.8.1 python-dotenv-0.13.0 python-raindropio-0.0.3 requests-2.24.0 six-1.15.0 urllib3-1.25.9
root@cf227dae855d:/# pip freeze
certifi==2020.4.5.2
chardet==3.0.4
idna==2.9
jashin==0.0.5
python-dateutil==2.8.1
python-dotenv==0.13.0
python-raindropio @ file:///python-raindropio/dist/python_raindropio-0.0.3-py3-none-any.whl
requests==2.24.0
six==1.15.0
urllib3==1.25.9
root@cf227dae855d:/# python
Python 3.8.3 (default, Jun  9 2020, 17:49:41) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import raindropio
>>> dir(raindropio)
['API', 'Access', 'AccessLevel', 'BrokenLevel', 'Collection', 'CollectionRef', 'DictModel', 'FontColor', 'Group', 'Raindrop', 'RaindropType', 'User', 'UserConfig', 'UserFiles', 'UserRef', 'View', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__', 'api', 'models']
>>> 
```

**Comment by drillan** [comment]
I checked the pypi wheel([python_raindropio-0.0.1-py3-none-any.whl](https://files.pythonhosted.org/packages/1d/af/8f22e875e7c9e78bd900b70e3046bedc41adaedd6ddf2c9dc5767e4637af/python_raindropio-0.0.1-py3-none-any.whl)) in pypi, and it looks like it doesn't contain [models.py](https://github.com/atsuoishimoto/python-raindropio/blob/master/raindropio/models.py).

**Comment by atsuoishimoto** [comment]
The version I released to PyPI is 0.0.1.  In this version, you should import API as follows.

```
from raindropio.api import API
```

Please see https://pypi.org/project/python-raindropio/ for sample.
